### ISSUE TITLE
Repository Name

### ISSUE THREAD

**Issue by hapily04** [skriptishot/JavaScript#1]
The current repository name is not descriptive of the projects goals whatsoever. Additionally, putting the language you're using in the repository is redundant as it already displays the most used repository. A name that would better fit the purpose of the repository'd contents would be "JSBotTemplate" or simply "BotTemplate".

**Comment by Iamlel** [comment]
I agree, the name of this repo should be changed, or the repo should be removed.
